### PR TITLE
[risk=no][RW-10342] Fix file path in recently-moved upload test

### PR DESCRIPTION
**Pull request by Peter-Lavigne**
Fix file path in recently-moved upload test. This test has been failing for a few days.

In https://github.com/all-of-us/workbench/pull/7802, this file was moved into the `nightly` directory. This broke a relative import.

---
**PR checklist**

- [x] I have included an issue ID or "no ticket" in the PR title as outlined in [CONTRIBUTING.md](https://github.com/all-of-us/workbench/blob/main/.github/CONTRIBUTING.md).
- [x] I have included a risk tag of the form `[risk=no|low|moderate|severe]` in the PR title as outlined in [CONTRIBUTING.md](https://github.com/all-of-us/workbench/blob/main/.github/CONTRIBUTING.md).
- [x] I have manually tested this change and my testing process is described above.
- [x] This change includes appropriate automated tests, and I have documented any behavior that cannot be tested with code.
- [x] I have added explanatory comments where the logic is not obvious.
- One or more of the following is true:
  - [ ] This change is intended to complete a JIRA story, so I have checked that all AC are met for that story.
  - [ ] This change fixes a bug, so I have ensured the steps to reproduce are in the Jira ticket or provided above.
  - [ ] This change impacts deployment safety (e.g. removing/altering APIs which are in use), so I have documented the impacts in the description.
  - [ ] This change includes a new feature flag, so I have created and linked new JIRA tickets to (a) turn on the feature flag and (b) remove it later.
  - [ ] This change modifies the UI, so I have taken screenshots or recordings of the new behavior and notified the PO and UX designer in [Slack](https://pmi-engteam.slack.com/archives/C02MWP2RN5P).
  - [ ] This change modifies API behavior, so I have run the relevant E2E tests locally because API changes are not covered by our PR checks.
  - [x] None of the above apply to this change.

### CODE REVIEW

**Review by Qi77Qi — APPROVED** [review]

File: e2e/tests/nightly/notebook/notebook-upload-file.spec.ts [hunk 1]
```diff
@@ -16,7 +16,7 @@ describe('Notebook Upload File Test', () => {
   const workspaceName = 'e2eTestNotebookUploadFile';
   const pyNotebookName = makeRandomName('Py3');
   const pyFileName = 'nbstripoutput-filter.py';
-  const pyFilePath = path.relative(process.cwd(), __dirname + `../../../resources/python-code/${pyFileName}`);
```
Comment (Qi77Qi) [inline e2e/tests/nightly/notebook/notebook-upload-file.spec.ts:19]: is it possible ot change this to use an absolute path?

Comment (Peter-Lavigne) [reply e2e/tests/nightly/notebook/notebook-upload-file.spec.ts:19]: Yeah, it should be. I went with the fastest fix for now, though.

Comment (Peter-Lavigne) [reply e2e/tests/nightly/notebook/notebook-upload-file.spec.ts:19]: I considered this but wanted to returned to [RW-10342](https://precisionmedicineinitiative.atlassian.net/browse/RW-10342) immediately - I think there are bigger E2E improvements to work on now 😄

Comment (Peter-Lavigne) [reply e2e/tests/nightly/notebook/notebook-upload-file.spec.ts:19]: I had considered this but wanted to returned to [the E2E spike](https://precisionmedicineinitiative.atlassian.net/browse/RW-10342) immediately - I think there are bigger E2E improvements to work on now 😄

Comment (Peter-Lavigne) [reply e2e/tests/nightly/notebook/notebook-upload-file.spec.ts:19]: I had considered this but wanted to return to [the E2E spike](https://precisionmedicineinitiative.atlassian.net/browse/RW-10342) immediately - I think there are bigger E2E improvements to work on.

**Pull request merged**